chrome/content/zotero/itemTree.jsx

```diff
@@ -3847,11 +3848,12 @@ var ItemTreeRow = function(ref, level, isOpen)
 
 ItemTreeRow.prototype.getField = function(field, unformatted)
 {
-	try {
+	// If the field is a standard field, use the standard method
+	// Note that being a standard field does not mean that it is a valid field for this item type
+	if (Zotero.ItemFields.getID(field)) {
```

Comment:
I don't think this would be right. Not all columns are valid item fields. We need to test for registered custom fields.

Reply:
Yeah. So the case is, if we test the column field to be a custom field or not, the custom columns data source would be the first choice, the default behavior, and ours be a fallback, which is strange. Our getField should be the default behavior I suppose.

Reply:
It's a single test one way or another, so there wouldn't be a performance difference from what you have here. If we just make sure it's fast it should be OK.

Reply:
Done